qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -56,6 +57,11 @@ setup()
         rbd --dest-pool ${POOL} --dest-namespace "${ns}" --no-progress import \
             ${DATA} ${IMAGE}
     done
+
+    # create another pool
+    ceph osd pool delete ${ANOTHER_POOL} ${ANOTHER_POOL} --yes-i-really-really-mean-it || true
```

Comment:
The script assumes no leftovers (see previous `rbd import`, etc), so no need to delete the pool.